specs/src/ante_handler_v3.md

```diff
@@ -14,6 +14,7 @@ The AnteHandler chains together several decorators to ensure the following crite
 - The tx's count of signatures <= the max number of signatures. The max number of signatures is [`TxSigLimit = 7`](https://github.com/cosmos/cosmos-sdk/blob/a429238fc267da88a8548bfebe0ba7fb28b82a13/x/auth/README.md?plain=1#L231).
 - The tx's [gas_limit](https://github.com/cosmos/cosmos-sdk/blob/22c28366466e64ebf0df1ce5bec8b1130523552c/proto/cosmos/tx/v1beta1/tx.proto#L211-L213) is > the gas consumed based on the tx's signatures.
 - The tx's [signatures](https://github.com/cosmos/cosmos-sdk/blob/22c28366466e64ebf0df1ce5bec8b1130523552c/types/tx/signing/signature.go#L10-L26) are valid. For each signature, ensure that the signature's sequence number (a.k.a nonce) matches the account sequence number of the signer.
+- The tx does not contain a MsgExec with a nested MsgExec or MsgPayForBlobs.
```

Comment:
should we backtick the msgs?

```suggestion
- The tx does not contain a `MsgExec` with a nested `MsgExec` or `MsgPayForBlobs`.
```

Reply:
Should this also include `MsgSubmitProposal`?

Reply:
Update: oh wait, we already have it [here](https://github.com/rootulp/celestia-app/blob/19ef71f1aa79c151c6ef8c1e8ea51d4ae74e5e12/specs/src/ante_handler_v4.md?plain=1#L18)